MagTag_Christmas_Countdown/code.py

```diff
@@ -0,0 +1,107 @@
+import time
+import board
+import displayio
+from adafruit_magtag.magtag import MagTag
+import adafruit_il0373
+from adafruit_display_shapes.circle import Circle
+
+#  create MagTag and connect to network
+magtag = MagTag()
+magtag.network.connect()
+
+#  setup display for use with displayio
+displayio.release_displays()
+display_bus = displayio.FourWire(board.SPI(), command=board.EPD_DC,
+                                 chip_select=board.EPD_CS,
+                                 reset=board.EPD_RESET, baudrate=1000000)
+time.sleep(1)
+display = adafruit_il0373.IL0373(
+    display_bus,
+    width=296,
+    height=128,
+    rotation=270,
+    black_bits_inverted=False,
+    color_bits_inverted=False,
+    grayscale=True,
+    refresh_time=1,
+    seconds_per_frame=1
+)
```

Comment:
Can you change to using the display from the MagTag library? `magtag.display`

Reply:
oh cool! i didn't know that you could do that if you were using `displayio` specific things. thanks!